helpers/localizePrice.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+const utils = require('handlebars-utils');
+
+const factory = () => {
+    return function(price, locale='fr') {
+        if (!utils.isObject(price) || !utils.isString(price.currency)
+            || isNaN(price.value) || !utils.isString(price.formatted)) {
+            // Return empty string if this does not appear to be a price object
+            return ''
+        }
+
+        if (!utils.isString(locale) || locale.length < 2) {
+            // Valid browser language strings are at least two characters
+            // https://www.metamodpro.com/browser-language-codes
+            // If provided locale is less than two characters (or not a string),
+            // return the normal formatted price
+            return price.formatted;
+        }
+
+        // Detect ICU support
+        const hasFullICU = (() => {
+            try {
+                const january = new Date(9e8);
```

Comment:
This is a little confusing. I'd prefer `new Date(0)` with a comment that this is 1/1/1970.

Reply:
This is lifted directly from docs: https://nodejs.org/api/intl.html#intl_detecting_internationalization_support